client/app/webUniverseComponents/charts/gaugeChart/gaugeChart.js

```diff
@@ -0,0 +1,102 @@
+import 'justgage';
```

Comment:
Does `JustGage` from ESLint global config could be removed? (https://github.com/ovh-ux/ovh-manager-web/blob/master/.eslintrc.json#L19)